packages/fetchai/contracts/oracle/contract.py

```diff
@@ -20,9 +20,11 @@
 """This module contains the class to connect to an Oracle contract."""
 
 import logging
-from typing import Any, Dict
+from typing import Any, Dict, cast
 
+from aea_ledger_cosmos import CosmosApi
```

Comment:
I'd remove cosmos api because this cosmwasm contract is not deployable on it in general.